### ISSUE TITLE
[Assistive tech and device support] Pages require sensory characteristics to be understood or operable. (11.11.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#32133]
#### VFS team name
Search & Discovery

#### VFS product name
Typeahead 2.0

#### Point of Contact/Reviewers
Josh Kim (Accessibility) @joshkimux

---

### Platform Issue
Pages require sensory characteristics to be understood or operable.

### Issue Details
Announcements to screen reader users that options are available and instructions on how to interact with suggestions are not available.

### Link, screenshot or steps to recreate
1. Using your choice of screen reader, type "10-10" into the staging search box. Confirm that there is no announcement. 
2. Repeat on va.gov. Confirm there is an announcement, but no additional instructions on search to enter.

### VA.gov Experience Standard
[Category Number 11, Issue Number 11](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.3_A

---

### Platform Recommendation
We recommend referencing gov.uk's usage of `aria-describedby` to provide additional "hints" on how to navigate and use the suggestions list (specifically, arrow keys and using enter to search). This should be separate from the announcement on how many suggestions exist when they expand out (x of x results).

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by shiragoodman** [comment]
@joshkimux could you validate please?

**Comment by joshkimux** [comment]
Can confirm! Closing out now
